server/channels/app/report.go

```diff
@@ -30,3 +26,12 @@ func (a *App) getUserReport(filter *model.UserReportOptions) ([]*model.UserRepor
 
 	return userReports, nil
 }
+
+func (a *App) GetUserCountForReport(filter *model.UserReportOptions) (*int64, *model.AppError) {
```

Comment:
Missing `filter.IsValid()` check as we're doing above in `GetUsersForReporting`.

Reply:
Same as above, `filter.IsValid()` assumes we have a sort column which we don't need for the user count.

Reply:
Actually, the isValid doesn't validate any field we're setting in count API, so its fine, my bad.